src/explore-education-statistics-common/src/modules/table-tool/components/DataTableCaption.tsx

```diff
@@ -30,10 +34,16 @@ export function generateTableTitle({
   }
 
   const locationsString = ` in ${commaList(
-    locations.map(location => location.label),
+    locations.length > 10 && !expanded
+      ? locations.slice(0, 10).map(location => location.label)
+      : locations.map(location => location.label),
   )}`;
 
-  return `Table showing '${subjectName}' from '${publicationName}'${locationsString}${timePeriodString}`;
+  return `Table showing '${subjectName}' from '${publicationName}'${
+    !expanded && locations.length > 10
+      ? `${locationsString} and ${locations.length} other locations...`
```

Comment:
`commaList` adds an 'and' so this will end up looking like 'Barnet and Barnsley and 5 other locations', which seems wrong.